cmake/VASTPackage.cmake

```diff
@@ -47,4 +45,45 @@ set(CPACK_DEBIAN_PACKAGE_CONTROL_EXTRA
     "${CMAKE_CURRENT_SOURCE_DIR}/scripts/debian/prerm")
 set(CPACK_DEBIAN_PACKAGE_DEBUG ON)
 
+# For the static binary builds it doesn't make much sense to install development
+# utilities, so we never do so.
+if (VAST_ENABLE_STATIC_EXECUTABLE)
+  get_cmake_property(CPACK_COMPONENTS_ALL COMPONENTS)
+  list(REMOVE_ITEM CPACK_COMPONENTS_ALL "Development" "Unspecified")
+endif ()
+
+# Create only a single package with all enabled install components rather than
+# creating one package per install component that potentially duplicates the
+# files. We should reconsider this in the future when we want to offer separate
+# runtime and development Debian packages.
```

Comment:
I don't understand how that would duplicate some of the files? Can't we just say we do this for simplicity but hope to change it in the future?

Reply:
It's because of the inter-package dependencies. If you install `Development` and that depends on `Runtime`, you install `Runtime` twice when you install both `Development` and `Runtime`. CPack is just stupid in that regard.

Reply:
I will simplify the comment somewhat, though.